source/billing.txt

```diff
@@ -46,33 +47,33 @@ how much you pay each month:
 
 .. note:: Track Your Usage
    
-   You can track an application's usage for the current month from the
+   You can track an application's usage for the current day from the
    application dashboard in the App Services UI.
 
-Monthly Free Tier
+Daily Free Tier
 ~~~~~~~~~~~~~~~~~
 
-All App Services Apps in a MongoDB Atlas project share a single monthly free
-tier. All usage below the free tier thresholds in a given month is not billed.
-As soon as a project exceeds any monthly free tier threshold, App Services starts 
+All App Services Apps in a MongoDB Atlas project share a single daily free
+tier. All usage below the free tier thresholds in a given day is not billed.
+As soon as a project exceeds any daily free tier threshold, App Services starts 
 billing for additional usage of any kind for that project.
```

Comment:
We evaluate billing on a daily basis, but what's the actual billing cycle for customers? They won't get a bill every day, right? We should let them know that their daily costs are rolled up into a monthly (or whatever) bill.

Unless they really do get a bill daily.

Reply:
Technical Reviewer, how often are customers sent a bill?

Reply:
It's a monthly invoice, so any day they go over the free tier limits will be a cost that is then summed at the end of the month. I think they also have day by day granularity if they want to check what their usage was on any specific day, but they will not be sent a bill daily.

Reply:
Added sentence about monthly invoice in the payment section.